examples/README.txt

```diff
@@ -0,0 +1,6 @@
+Examples Gallery
+================
```

Comment:
Is it possible to add a new example in the gallery, comparing SVM and QSVM on toy datasets?
You can start from this code: https://scikit-learn.org/stable/auto_examples/classification/plot_classifier_comparison.html

Reply:
Yes, of course. Do you prefer me to implement it in this PR or in a separate issue?

Reply:
Yes, you can add the example in this PR.